src/lightbox/LightboxFooter.js

```diff
@@ -39,7 +41,11 @@ export default class LightboxFooter extends PureComponent<Props> {
       <SafeAreaView mode="padding" edges={['right', 'bottom', 'left']}>
         <View style={[styles.wrapper, style]}>
           <Text style={styles.text}>{displayMessage}</Text>
-          <Icon style={styles.icon} color="white" name="more-vertical" onPress={onOptionsPress} />
+          <Pressable style={styles.iconTouchTarget} onPress={onOptionsPress} hitSlop={10}>
```

Comment:
Neither button was giving the touch target the minimum 48 x 48 px
    size recommended by Material Design [1]; so, do that with the
    `hitSlop` prop [2]. Passing one number like this adds that amount of
    additional distance outside the element in which a press can be
    detected. So, with a font size of 28, the new width and height of
    the pressable area are approx. 10 + 28 + 10 = 48.

One thing I see in that handy Pressable doc is this:

> The touch area never extends past the parent view bounds and the Z-index of sibling views always takes precedence if a touch hits two overlapping views.

So this `hitSlop` will only take effect to the extent that there's that much room around this view within its parent view. Is that the case here? We might need to arrange some margin here, or padding in the parent, to make sure of that.

Reply:
Ah, I think your screenshot in the comment above may answer this question. It looks like we're probably in good shape except for the top side of the three-dots button.

It'd be good to add some padding or margin to give that target its full size, and perhaps also to structure things so that it's clear they both do. Looking at the header code, I think that one is getting enough room only because the avatar holds up the header's height. Perhaps add a `margin` to each `Pressable`'s style, equal to its `hitSlop`?

Reply:
> Perhaps add a `margin` to each `Pressable`'s style, equal to its `hitSlop`?

![image](https://user-images.githubusercontent.com/22248748/106332434-d083b700-6254-11eb-881c-44509bc75765.png)

Hmm, this feels like too much vertical space. The `margin` I've added gets added to the `paddingVertical: 8` in `styles.wrapper`.

Reply:
(I suppose the same is happening with the horizontal padding, too.)

Reply:
Ah, indeed.

I think it'd be fine to add margin that only supplements the parent's padding. We want 12 total space on each side, so where the parent's padding is 16 that's enough by itself, but where it's 8 we want a margin of 4.